website/docs/content/writing-for-ui-copy/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Writing for UI Copy
+caption: Applications should be usable and accessible for all users of differing abilities.
+description: Applications should be usable and accessible for all users of differing abilities.
+previewImage: assets/illustrations/foundations/accessibility.jpg
+---
```

Comment:
**question:** @majedelass Is it intended for this page to be visible on the site right now?

Reply:
Nope!! this was my bad, I rebased wrong and accidentally added them back. Theyre removed again.